scripts/scil_bundle_screenshot_mni.py

```diff
@@ -50,13 +51,14 @@ def _build_arg_parser():
     p.add_argument('in_anat',
                    help='Path of the reference file (.nii or nii.gz).')
     p.add_argument('--target_template',
-                   help='Path to the target MNI152 template for registration. \n'
-                        'If in_anat has a skull, select a MNI152 template \n'
+                   help='Path to the target MNI152 template for registration. '
```

Comment:
Why did you remove all the \n? We don't have a real rule about this, but we try to limit the lengths.

Reply:
Ouh my, I did not saw that :open_mouth: I made a lot of docstring/cli changes using Copilot and maybe overlooked some of them. I'll go through the helps and fix them.

Reply:
Maybe I open a pandora's box here. Going through scripts, this convention is really weakly enforced, in particular in newer scripts. It will be long going through all the scripts to fix this and I don't see the point doing it if we're to open an issue or do the work aside.

However, there is a PR in its way to add this kind of functionality in core python : python/cpython#24377. It seems it took a long time to get there, but looks like it's near completion. We should track that, I will open an issue about it. Just learned that there is also a formatter for defaults, would have saved us some time ...